dashboard-for-web/articles/web-dashboard-designer-mode/ui-elements/dialogs-and-wizards/query-builder.md

```diff
@@ -67,9 +67,9 @@ The following settings are available for each column.
 ## Filter Data
 The Query Builder allows you to filter a query. To do this, deselect tables and click the ellipsis button in the appeared **Filter** field in the **Properties** pane. This invokes the **Filter Editor** dialog, which provides a visual interface for constructing a filter string.
 
-![wdd-filter-editor-filter-query](../../../images/img124905.png)
+![wdd-filter-editor-filter-query](../../../../images/img124905.png)
 
-To learn more, see [Filter Queries](working-with-sql-data-sources/filter-queries.md).
+To learn more, see [Filter Queries](../../provide-data/working-with-sql-data-sources/filter-queries.md).
 
```

Comment:
Summary for this MR:
Overall, it is well-written. Here are a few points to focus on in future documents:
- Please review article use (an EF data source, a Federated data source, the number of non-null objects). Also, when talking about a "mode" (for example, _advanced_ mode), note that the article is not necessary.
- Also, try to decrease the number of gerunds used.

##GrammarReview